src/status_im/mailserver/core.cljs

```diff
@@ -37,9 +37,13 @@
 (def seven-days (* 7 one-day))
 (def maximum-number-of-attempts 2)
 (def request-timeout 30)
+(def min-limit 200)
+(def max-limit 2000)
+(def default-limit max-limit)
 (def connection-timeout
   "Time after which mailserver connection is considered to have failed"
   10000)
+(def limit (atom default-limit))
```

Comment:
just wondering, why do you use atom and not app-db?

Reply:
I didn't think much of it, I instinctively went for an atom as it's very much local state, not used for subscriptions etc, rather than global (no-one should be accessing this from anywhere), but I don't mind putting in re-frame db, it's all good with me